Twilio-org/phonebank
# [Ticket 135] Make client detect no calls available and only let user stop calling.

Manual testing:
1. modify your seed file temporarily, changing line 60 to
```javascript
const contactParams = new Array(1).fill(null).map(() => {
```
2. run `npm run db:refresh` from you command line
3. run the server, register as a user and join campaign 1.
4. Then, click the "start calling" button. Go to the calls page and then refresh the page without performing further actions. This keeps the call in assigned status.
5. Click "start calling" again and repeat step 3 once more as there should be one more call to 'stick' in assigned status.
6. Run `npm run db` from your command line and `select * from calls;` to see all call entries, and verify that they all have status 'assigned'
7. Go back to your browser and click "start calling" once more, this time when the calls page loads note the following outcome:
- the top of the sidebar said "No Calls Available"
- the form and all buttons except "stop calling" are disabled.
